packages/keyring/src/Keyring.ts

```diff
@@ -534,4 +540,19 @@ export default class Keyring {
       }
     }
   }
+
+  async verifyAccountBackupPassword(backup: AccountBackup, password: string) {
```

Comment:
We can probably reuse this method inside `importAccount` to remove the duplication of verifying backup password.
```suggestion
  async verifyAccountBackupPassword(backup: AccountBackup, password: string): Promise<KeyringPair> {
```